nltk/parse/dependencygraph.py

```diff
@@ -49,14 +60,11 @@ def __init__(self, tree_str=None, cell_extractor=None, zero_based=False, cell_se
         are split by whitespace.
 
         """
-        self.nodes = defaultdict(lambda: {'deps': defaultdict(list)})
+        self.nodes = defaultdict(lambda: deepcopy(DependencyGraph.default_node))
```

Comment:
So what about to declare ``DependencyGraph.get_default_node()``which for now will return the default node with all the keys?